docs/ossm/ambient/waypoint.md

```diff
@@ -0,0 +1,233 @@
+# Getting Started with Istio Waypoint in Ambient Mode (Tech Preview)
+
+This section provides a high-level overview and installation procedure for Istio's ambient mode on OpenShift Container Platform (OCP) using OpenShift Service Mesh 3.1.x.
+
+---
+
+## 1. Overview of Istio Ambient Mode
+
+Istio's ambient mode offers a sidecar-less approach to service mesh, simplifying operations and reducing resource consumption. Instead of injecting a sidecar proxy into every application pod, ambient mode utilizes a node-level **ZTunnel proxy** for secure, mTLS-enabled connections and an optional **Waypoint proxy** for advanced L7 functionalities.
+
+### 1.2 Why Use Istio's Ambient Mode?
+
+Istio's ambient mode offers several benefits:
+
+* **Simplified Operations:** Eliminates the need for sidecar injection and management, reducing operational complexity and cognitive load.
+* **Reduced Resource Consumption:** By centralizing mTLS and L4 policy enforcement in the `ztunnel`, ambient mode significantly lowers resource overhead per pod.
+* **Incremental Adoption:** Allows for gradual adoption of service mesh features. Workloads can join the mesh at L4 for mTLS and basic policy, and then selectively opt-in for L7 features via `waypoint` proxies as needed.
+* **Enhanced Security:** Provides a secure, zero-trust network foundation with mTLS by default for all meshed workloads.
+
+**Trade-offs:**
+
+* Ambient mode is a newer architecture and may have different operational considerations compared to the traditional sidecar model.
+* L7 features require the deployment of `waypoint` proxies, which add a small amount of overhead for the services that utilize them.
+
+---
+
+## 2. Pre-requisites to Using Waypoint Ambient Mode with OSSM 3
+
+Before installing Istio's ambient mode with OpenShift Service Mesh, ensure the following prerequisites are met:
+
+* **OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
+* **OpenShift Service Mesh 3.1.0+ operator is installed:** Ensure that the OSSM operator version 3.1.0 or later is installed on your OpenShift cluster.
+* **Istio deployed in Ambient Mode:** Refer to OSSM Ambient [initial doc](README.md).
```

Comment:
The last bullet only mentions Istio deployed in ambient mode, should there also be a pre-requisite to mention that the workloads involved are part of the mesh in ambient mode? It could be that Istio is deployed in ambient mode, but the workloads/namespaces don't have the appropriate labels for Ztunnel traffic redirection.

Reply:
Added.